.circleci/config.yml

```diff
@@ -762,6 +790,34 @@ jobs:
                 variant: "debug"
       - run: exit 0
 
+  billing-tests:
```

Comment:
Maybe build tests without running on each PR, so that we know that e2e tests could be compiled?

Reply:
Hmm, this will increase CI time. 
I think it's not really needed given that we have a stable API and we run these tests daily. So if anything is broken, we'll know it within a day.